### ISSUE TITLE
Signup/Login not possible due to JSON error.

### ISSUE THREAD

**Issue by cosminroger** [Supernova-Studio/public-issue-tracker#181]
Hi. I am having trouble when trying to register or login to the app. Below you can find a screenshot with the error. At the register part, I have tried both the normal account creation and also the google signup. Neither works. I hope it will get fixed soon.

Screenshot:
<img width="425" alt="Error" src="https://user-images.githubusercontent.com/38468551/74610800-a3e39780-50f6-11ea-9af8-68750287a08b.png">

**Comment by jan-supernova** [comment]
Hey cosminroger,
Are you using the latest V8.4 version of the Supernova Studio? We sorted most of the issues with JSON errors in the V8.3, but we are still working on squashing all of the remaining issues regarding Logging In/Signing Up process.
Another thing you can try is to go to https://app.supernova.io/, click on "Don't have an account? Sign Up," create your account there and then log into it in the Supernova Studio. 
Let me know if this solved your issue. If not, we will take a closer look at it. 

Have a wonderful day!
Jan, Supernova Team

**Comment by cosminroger** [comment]
Hi. Thanks for the reply. Signing up on the link provided doesn't work either. I get a "Network Error" message. Also, I am using whatever version is currently provided through the [https://supernova.io/](https://supernova.io/). I don't know where I can see what version that is.

**Comment by jan-supernova** [comment]
Hey!
The Studio version can be checked by launching the Studio, selecting `Supernova Studio` in the top menu on your Mac and clicking on `About Supernova Studio`. It is not necessary to be logged in for this.
![Screenshot 2020-02-18 at 09.22.05.png](https://images.zenhubusercontent.com/5d13751984dbbd27942b5cd3/2503b978-4029-4773-9e8d-af6aefe37ef2)
However, since you let us know that registering on Cloud didn’t work for you either, we are currently checking for the possible cause of the `Network Error` you had on Cloud, since those issues are definitely going to be caused by the same issue. 
Registering on Cloud should always work, because it is a web app. For now, it appears to be some sort of an issue with Firebase. We will let you know when we have any news regarding your issue.

Have a wonderful day!
Jan, Supernova Team

**Comment by jan-supernova** [comment]
Hey cosminroger,
We would like to inform you that this issue should be resolved by now. Can you try to register and login again and let us know if the process works properly for you now? Thank you for letting us know!

Have a wonderful day!
Jan, Supernova Team

**Comment by cosminroger** [comment]
Hey jan,

Yes, now it works. I have tried to go onto **app.supernove.io**  and I logged in automatically from yesterday's attempt of registration. Because of this, I have tried to login through the mac app as well and it worked. Thank you for your time! About the version of the app I was using, it was the latest. I have just check that with the one from the previous post.

Have a nice day and good luck with the project,
Cosmin Roger Harsulescu
